### DISCUSSION TITLE
Could Cowasm run inside a Stackblitz-webcontainer?

### DISCUSSION THREAD

**Discussion by martin12333** [sagemathinc/cowasm#57] [Q&A]
If I understand correctly, Cowasm , when being run from nodejs, is **_Js+wasm, no native-code_**  files, that's important.

// I plan to study and do experiments

( https://github.com/stackblitz/webcontainer-core )

**Comment by williamstein** [comment]
> is Js+wasm, no native-code files, that's important.

It optionally uses a module called `posix-node` which I also wrote (in Zig) to expose a lot of extra posix functionality, e.g., the networking stack (which is used by python so far).   https://www.npmjs.com/package/posix-node

However, if posix-node isn't available, then cowasm will still run from nodejs, and I think there are no other native code dependencies.
